pkg/oc/cli/cancelbuild/cancelbuild.go

```diff
@@ -210,7 +210,7 @@ func (o *CancelBuildOptions) RunCancelBuild() error {
 			}
 		}
 
-		if stateMatch && !buildutil.IsTerminalPhase(build.Phase) {
+		if stateMatch && !buildutil.IsTerminalPhase(build.Status.Phase) {
```

Comment:
sneaky :-)

Reply:
Not sneaky, but that's how the external API differs from internal.